src/Catalyst.Core.Modules.Consensus/Deltas/DeltaCache.cs

```diff
@@ -55,6 +55,8 @@ public class DeltaCache : IDeltaCache, IDisposable
 
         public static Address TruffleTestAccount = new Address("0xb77aec9f59f9d6f39793289a09aea871932619ed");
 
+        public static Address CatalystTruffleTestAccount = new Address("0x58BeB247771F0B6f87AA099af479aF767CcC0F00");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=5051857)